ferrocene/doc/specification/src/attributes.rst

```diff
@@ -503,7 +503,9 @@ Attribute ``target_feature``
 .. rubric:: Legality Rules
 
 :dp:`fls_3qj3jvmtxvx6`
-:t:`Attribute` :c:`target_feature` shall apply to :t:`[unsafe function]s`.
+Safe :t:`[function]s` that are annotated with :c:`[target_feature]s`
+can only be called without an :t:`unsafe block` by a caller that is within a function
+that enables all the :c:`[target_feature]s` that the callee enables.
```

Comment:
```suggestion
Safe :t:`[function]s` that are subject to :t:`attribute` :c:`target_feature` shall be called within an :t:`unsafe context` unless it is called within a :t:`function` subject to the same set of :c:`[target_feature]`s.
```

(not sure if `unsafe context` is a thing, please look at how other unsafe operations describe their restrictions)